docs/glossary.rst

```diff
@@ -225,6 +225,32 @@ Check Multiple Choice
                             "That's a clown who likes burgers.",
                             "Correct! Head over to the next exercise!"])
 
+Recommended approach for testing train test splits
```

Comment:
I would add a link to [this section](https://pythonwhat.readthedocs.io/en/latest/articles/checking_function_calls.html?highlight=kwargs#signatures).

That section should mention that the names of these arguments are not fixed, something like:

>Note that the `*args` argument, while common does not need to be named `args`. Pythonwhat expexts the real argument name. Use `inspect.signature` as done above to know the argument name.

It could link to [this section](https://github.com/datacamp/content-wiki/blob/master/docs/content-engineering/sct-workflow.md#find-the-real-signature) about how to get the function signature.